config/locales/validators/en.yml

```diff
@@ -6,3 +6,4 @@ en:
       format:
         allow_blank: should look like an email address. Please use a different address or leave blank.
         no_blank: should look like an email address.
+    forbidden_name: "%{value} is not allowed"
```

Comment:
Not AD&T but would appending "as a user name" add more clarity to the error message?

Reply:
When Rails displays the errors, it will prepend the field name (that's why I overrode `login` to be `User name` instead). Here's an example of how it looks on the sign-up page:
![an error message with the header 'Sorry! We couldn't save this user because:' and text 'User name foobar is not allowed' inside a light red rectangle](https://github.com/otwcode/otwarchive/assets/13002992/1fa06907-e62a-4bf5-810e-df3246f29e5d)

and for the guest comment form:
![an error message with the header 'Sorry! We couldn't save this comment because:' and the text 'Name foobar is not allowed' inside a light red rectangle](https://github.com/otwcode/otwarchive/assets/13002992/1065b338-f265-40eb-bfd2-9f36496022d5)

That being said, @sarken do we have any preference from PAC on the exact copy for this?

Reply:
I'll check with PAC and Support!

Reply:
Neither PAC nor Support had objections to this wording.